components/automate-ui/src/app/modules/team/team-details/team-details.component.ts

```diff
@@ -152,7 +152,8 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
     combineLatest([this.isIAMv2$, team$]).pipe(
       takeUntil(this.isDestroyed),
       filter(([_isV2, team]) => team !== undefined),
-      distinctUntilChanged(([isV2, team]) => isV2 === isV2 && team === team)
+      distinctUntilChanged(([isV2A, teamA], [isV2B, teamB]) =>
```

Comment:
`lastTeam` and `currentTeam` maybe?

Reply:
mmm yessss

Reply:
it's almost like we want `previousIAMVersion` and `previousTeamType`